include/nbl/asset/filters/CBlitImageFilter.h

```diff
@@ -152,6 +164,89 @@ class CBlitImageFilter : public CImageFilter<CBlitImageFilter<Swizzle,Dither,Nor
 					return CScaledImageFilterKernel<Kernel>(fScale,kernel);
 				}
 
+				static bool computePhaseSupportLUT(const CState* state)
```

Comment:
I believe you want to move the core of this to the `CBlitUtilities`

Reply:
Yep you're right.